.gitlab-ci.yml

```diff
@@ -4,12 +4,17 @@ stages:
   - mirror
 
 mirror-from-github:
+    retry: 2
     stage: mirror
     rules:
           - if: $TOKEN
     before_script:
         - source /etc/os-release && zypper ar http://download.suse.de/ibs/SUSE:/CA/openSUSE_Leap_${VERSION}/SUSE:CA.repo
-        - zypper -n in git ca-certificates-suse
+        - |
+          for i in {1..3}; do
+              zypper -n in git ca-certificates-suse && zypper ref && break
```

Comment:
So the refresh is only done when the install passes?

Reply:
yeah, and then if refresh passes -> loop is stooped. It pretty clever solution , only `--force` in refresh will be nicer

Reply:
but we should only need to refresh if the install *fails*, right?

Reply:
Are there any plans to fix this? Is there a ticket?

Reply:
I also wasn't sure about this. That's why I proposed it as alternative in my previous comment (when requesting changes).